package.json

```diff
@@ -11,12 +11,12 @@
     "web": "expo start --web-only --web"
   },
   "dependencies": {
-    "react": "16.13.1",
-    "react-dom": "^16.13.1",
-    "react-native": "0.63.1",
-    "react-native-gesture-handler": "^1.6.1",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-native": "^0.63.3",
+    "react-native-gesture-handler": "^1.8.0",
     "react-native-reanimated": "2.0.0-alpha.8",
-    "react-native-web": "^0.12.3"
+    "react-native-web": "^0.14.7"
```

Comment:
we cannot upgrade this package due to [this issue](https://github.com/software-mansion/react-native-gesture-handler/issues/1164#issuecomment-684013868)

Please leave it as it is along with `react-dom`(otherwise there are errors)